src/views/docs/en/guides/extend/plugins.md

```diff
@@ -205,15 +297,13 @@ All arguments arrive as a bag of options with the following properties:
 
 ## Helper Methods for Plugin Authors
 
-For common Architect Plugin use cases, Architect provides a few helper functions available in core Architect packages to make life easier for plugin authors.
+For common Architect Plugin use cases, Architect provides a few helper functions available as parameters injected as arguments into plugin methods to make life easier for plugin authors.
 
-### `createLambdaJSON`
+### `createFunction`
 
-> `createLambdaJSON({ inventory, src })`
+> `createFunction({ inventory, src })`
 
-Available in the [`@architect/package`](https://npmjs.com/package/@architect/package) module, this method can be leveraged inside a plugin's [`package`](#package) method in order to easily and consistently define CloudFormation JSON representing Lambdas created by the plugin.
-
-Leveraging this helper method gives the plugin function support for [arc's per-function runtime configuration via the `config.arc` file](https://arc.codes/docs/en/reference/config.arc/aws) transparently.
+This method should be leveraged inside a plugin's [`package`](#package) method in order to more easily define CloudFormation JSON representing Lambdas created by the plugin. Use of this method for defining Lambdas is an Architect best practice as certain specific conventions that Architect relies on can be maintained. While the AWS Lambda logical ID is generally not a concern for developers using Architect, Architect relies on a logical ID naming convention to e.g. retrieve execution logs of a deployed Lambda via [`arc logs`][logs]. This helper method helps enforce such conventions. Leveraging this helper method also gives the plugin function transparent support for [arc's per-function runtime configuration via the `config.arc` file](https://arc.codes/docs/en/reference/config.arc/aws); as the plugin author you do not need to worry about parsing each individual `config.arc` file and retrieving and setting its configuration options (such as assigned RAM, concurrency maximums, and timeout limits).
```

Comment:
Can we break this up a bit? It's just a lot.

Also to differentiate between filenames etc., stylistically strongly prefer `Architect` over `arc`, personally.